server/server.go

```diff
@@ -180,6 +180,8 @@ func (s *Server) Start(ctx context.Context) error {
 	)
 
 	api.RegisterPoetServiceServer(grpcServer, rpcServer)
+	api.RegisterConfigurationServiceServer(grpcServer, rpcServer)
```

Comment:
The configuration service should probably not be exposed on the same, publicly available port as the public API.

Reply:
Done